dietpi/func/dietpi-globals

```diff
@@ -2111,6 +2111,15 @@ could not be found in file \$3
 
 	}
 
+	# Subprocess-less sleep
+	# - $1 = Number of seconds to pause. Can be a fractional number.
+	G_SLEEP()
+	{
+		[[ $_snore_fd ]] || exec {_snore_fd}<> <(:)
+		read -rt "$1" -u "$_snore_fd" || :
+	}
+
```

Comment:
Let's initialise the file descriptor variable as empty string when globals are loaded, and use a name that is related to the function 🙂.
```suggestion
	G_SLEEP_FD=
	G_SLEEP()
	{
		[[ $G_SLEEP_FD ]] || exec {G_SLEEP_FD}<> <(:)
		read -rt "$1" -u "$_snore_fd" || :
	}
```

Reply:
Done!